src/generics/cacheHelper.js

```
@@ -0,0 +1,1899 @@
+					const usersFromDb = await userQueries.getUsersByUserIds(missingUserIds, {}, tenantCode, false)
+
+					if (usersFromDb && usersFromDb.length > 0) {
+						// Add fetched users to result
+						dbFetchedUsers.push(...usersFromDb)
+
+						// Create user ID to is_mentor mapping using map
+						userMentorEntries = usersFromDb.map((user) => [
+							user.user_id,
+							{
+								is_mentor: user.is_mentor || false,
+								organization_code: user.organization_code || user.organization_id,
```

Biome (2.1.2)

[error] Can't assign userMentorEntries because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)